src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -40,8 +40,10 @@ public class AddCommand extends Command {
             + PREFIX_PHONE + "98765432 "
             + PREFIX_EMAIL + "johnd@example.com "
             + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
+            + PREFIX_DATE_OF_BIRTH + "9 Mar 1999"
```

Comment:
Might want to consider standardising across the board the way user inputs date so that it is the same for dob and scheduling